packages/Frontend/src/components/View.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
```

Comment:
Let's try wrapping all of the routes except the home route in this view component.